cogs/search_utils/regex.py

```diff
@@ -0,0 +1,150 @@
+from re import compile, I, X
+
+nono_re = compile(
```

Comment:
Also update all references to this.
```suggestion
filter_words = compile(
```